regparser/tree/reg_text.py

```diff
@@ -84,6 +85,95 @@ def build_subpart(text, part):
         "", [], label, subpart_title, node_type=struct.Node.SUBPART)
 
 
+def subjgrp_label(starting_text, letter_list):
+    words = starting_text.split(" ")
+    candidate_text = ""
+    suffixes = [""] + [_ for _ in string.lowercase[1:]]
```

Comment:
Why skip `a`?

Also, not sure if you'd like this more:
`list(string.lowercase[1:])`